app/javascript/packs/ping_things/ping_thing_table.vue

```diff
@@ -30,24 +27,24 @@
           </select>
         </div>
 
-        <button @click.prevent="get_filtered_records()"
+        <button @click.prevent="get_records()"
```

Comment:
since there are no keyups, is it possible to trigger search after hitting enter?